index.html

```diff
@@ -0,0 +1,82 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <link rel =" stylesheet" href="Style.css" />
+  <title>My CV</title>
+</head>
+<body class="body_class">
+  <header>
+    <h1 class="h1_class">CV</h1>
+  </header>
+  <nav>
+      <ul class = "nav_class">
+        <li class="li_class"><a href = "#Name" class="a_to_li"><article>Name</article></a></li>
+        <li class="li_class"><a href="#Summary" class="a_to_li"><article>Summary</article></a></li>
+        <li class="li_class"><a href="#Skills" class="a_to_li"><article>Skills</article></a></li>
+        <li class="li_class"><a href="#Experience" class="a_to_li"><article>Experience</article></a></li>
+        <li class="li_class"><a href="#Education" class="a_to_li"><article>Education</article></a></li>
+        <li class="li_class"><a href="#English_level" class="a_to_li"><article>English level</article></a></li>
+      </ul>
+  </nav>
+  <main>
+      <section>
+          <img src="Id5KvUU7lG0[1].jpg" class="img_class" alt = "My Photo"/>
+      </section>
+      <section class = "section_class" id="Name">
+        <article class="article_class">
+          <h4 class="h4_class">Name:</h4> <span class = "span_class_info"> Egor</span>
+        </article>
+        <article>
+            <h4 class="h4_class">Surname:</h4> <span class="span_class_info"> Popov</span>
+        </article>
+      </section>
+      <section class="section_class" id="Summary">
```

Comment:
Please, read about semantics of "section". We don't need to use section every time, when we need block :)